angular-ngc/tools/BUILD.bazel

```diff
@@ -21,6 +21,7 @@ js_binary(
 ts_project(
     name = "ngc.esbuild",
     srcs = ["ngc.esbuild.ts"],
+    supports_workers = False,
```

Comment:
?

Reply:
Without this one, it will report error in my local...

Reply:
That doesn't make sense...

Reply:
I am using `WSL 2` under Windows, is that the reason?

Reply:
That shouldn't require any workarounds, especially in an example...

Reply:
FYI: https://github.com/aspect-build/rules_ts/commit/b773287bdde683f1c993c7d981d6ecd19f6d1bb7

Reply:
You can revert this after rebasing

Reply:
done.